HISTORY.md

```diff
@@ -6,15 +6,34 @@ dev
 
 - \[Short description of non-trivial change.\]
 
+
+2.28.0 (2022-06-08)
```

Comment:
Remember to update this if you don't release today.

Reply:
Yep, I'll move it to tomorrow. No ones getting paged tonight 😁